components/hal_drv/bl602_hal/hal_board.c

```diff
@@ -726,6 +726,7 @@ static int hal_board_load_fdt_info(const void *dtb)
         } else {
             pwr_table_ble = 0;
         }
+        pwr_table_ble += 0;  //  Suppress "variable set but not used" warning
```

Comment:
Couldn't a #pragma be used here instead, just wondering?

Reply:
Sorry haven't used #pragma. Lemme do some research

Reply:
Reverting the fix for "variable set but not used" warning. I think we need to fix the upstream code to suppress warnings for variables like `pwr_table_ble` that are set but not used when logging is disabled.

Meanwhile, Mynewt compilation with BL602 HAL will fail until we find a proper fix for this.